packages/jupyterlab-chat/src/factory.ts

```diff
@@ -130,6 +135,7 @@ export namespace ChatWidgetFactory {
     chatCommandRegistry?: IChatCommandRegistry;
     attachmentOpenerRegistry?: IAttachmentOpenerRegistry;
     inputToolbarFactory?: IInputToolbarRegistryFactory;
+    chatFooterRegistry?: IChatFooterRegistry;
```

Comment:
We should probably reuse the `IChatBodyProps` from `@jupyter/chat` instead of listing all of the properties here.